### ISSUE TITLE
Accessibility: Pagination list is not accessible by keyboard

### ISSUE THREAD

**Issue by zhr1130** [react-bootstrap-table/react-bootstrap-table2#1314]
**Describe the bug**
Rows per page list is not accessible by using keyboard. The link somehow has tabindex = -1 .

**To Reproduce**
1. Go to basic pagination table example in storybook
2. Click rows per page button. 
3. List is not accessible by keyboard due to <a> has tabindex = -1.

**Comment by AllenFang** [comment]
@zhr1130 I will handle it. thanks!

**Comment by AllenFang** [comment]
@zhr1130 I guess this requirement is little bit difficult to implement, because bootstrap dropdown will be collapse when you press any other key(Include tabs). if you have any other online case for this usage, please let me know. thanks

**Comment by zhr1130** [comment]
Hi Allen, 

Thanks for the reply. I don't have other usage, we are using it for our product and it's a great library, however we have to meet accessibility bar which required all elements are accessible by keyboard. Do you have any suggestion that i can hack my own code to achieve that ?

**Comment by AllenFang** [comment]
@zhr1130 maybe you have to custom the size per page dropdown
please read the docs:

it's a advance customization
https://react-bootstrap-table.github.io/react-bootstrap-table2/docs/basic-pagination.html#advance-customization

You can also custom dropdown literally by standalone feature:
https://react-bootstrap-table.github.io/react-bootstrap-table2/docs/basic-pagination.html#completely-customization

We also have a lots of online example
https://react-bootstrap-table.github.io/react-bootstrap-table2/storybook/index.html

Please open another issue if you have any issues or questions, thanks
